packages/plugin-flex/src/prints/deploy.ts

```diff
@@ -0,0 +1,44 @@
+import { Logger, singleLineString, boxen, coloredStrings } from 'flex-dev-utils';
```

Comment:
do you need to update the `flex-dev-utils` imports to be relative or does this still work?

Reply:
https://github.com/twilio/flex-plugin-builder/pull/389/files#diff-741b7fc253609e1baa66e564fe61465771fa020560b1f31ee6c19ed9a792223c